moduleroot/.travis.yml.erb

```diff
@@ -106,11 +106,34 @@ matrix:
       <%= key %>: <%= job[key].gsub(/@@SET@@/, set['set']).gsub(/@@COLLECTION@@/, set.fetch('collection', 'puppet6')).gsub(/@@TESTMODE@@/, set.fetch('testmode', 'apply')) %>
 <%   end -%>
 <% end -%>
+<% if @configs['use_litmus']['enabled']
+  @configs['use_litmus']['puppet_collection']||=['puppet5', 'puppet6']
+  @configs['use_litmus']['provision_list']||=['travis_el','travis_deb']
+  @configs['use_litmus']['rvm']||='2.5.3'
```

Comment:
nit: should be a space between the assignment operator there `@configs['use_litmus']['rvm'] ||= '2.5.3'`

Reply:
i'll update it
Cheers!